### ISSUE TITLE
The attention mask and the pad token id were not set

### ISSUE THREAD

**Issue by Lelege0** [neemiasbsilva/MiniGPT4-image-caption-generation#1]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.
请问这个怎么解决？
